Makefile

```diff
@@ -140,22 +160,70 @@ rm -rf $$TMP_DIR ;\
 }
 endef
 
-# Generate bundle manifests and metadata, then validate generated files.
 .PHONY: bundle
-bundle: manifests kustomize
+bundle: manifests kustomize ## Generate bundle manifests and metadata, then validate generated files.
 	operator-sdk generate kustomize manifests -q
```

Comment:
The operator-sdk should be pinned to a version. There should be a target to install the tool in `./bin` and used from there.

Reply:
It's using v1.15.0, i agree though we should just put the version we use in the bin directory so it's pinned. This is just how the bog standard generated Makefile looks. Can look into pinning the sdk version separately.

Reply:
We've pinned the version of the oprator-sdk binary in other repos, but since we are no longer generating a bundle for the controller, and it's the only make target that uses it, it's probably not needed here.